lib/Support/Unix/Signals.inc

```diff
@@ -264,13 +247,6 @@ void llvm::sys::DontRemoveFileOnSignal(StringRef Filename) {
   std::vector<std::string>::iterator I = FilesToRemove->end();
   if (RI != FilesToRemove->rend())
     I = FilesToRemove->erase(RI.base()-1);
```

Comment:
The reason for iterator `I` is to enable the loop below - without the loop, you don't need `I` at all.